### DISCUSSION TITLE
Missing column name in error "Unexpected null for non-null column"

### DISCUSSION THREAD

**Discussion by qarmin** [diesel-rs/diesel#3798] [Ideas]
Missing table/column name not really helps to find exact root of problem.

Can be name of table/column be added to this and similar errors?

**Comment by weiznich** [comment]
It's unfortunately not as simple as "just add the table/column" name there where this errors are generates, as these locations do not have access to the relevant information. 
https://github.com/diesel-rs/diesel/blob/d4a8a6738fbfe3a00ecb16527db3f507edc2f464/diesel/src/deserialize.rs#L443

That written: If someone is willing on designing and implementing a change which includes these information in a non breaking way I'm willing to review a PR for that.
